repo/content/content_manager_indexes.go

```diff
@@ -31,16 +31,16 @@ func (bm *Manager) CompactIndexes(ctx context.Context, opt CompactOptions) error
 		return errors.Wrap(err, "error loading indexes")
 	}
 
-	contentsToCompact := bm.getContentsToCompact(ctx, indexBlobs, opt)
+	blobsToCompact := bm.getBlobsToCompact(ctx, indexBlobs, opt)
 
-	if err := bm.compactAndDeleteIndexBlobs(ctx, contentsToCompact, opt); err != nil {
+	if err := bm.compactAndDeleteIndexBlobs(ctx, blobsToCompact, opt); err != nil {
 		log(ctx).Warningf("error performing quick compaction: %v", err)
 	}
 
-	return nil
+	return bm.indexBlobManager.cleanup(ctx)
```

Comment:
Doesn't `compactAndDeleteIndexBlobs` call `registerCompaction` => `deleteOldBlobs` => `cleanupCleanupBlobs`?
Wouldn't adding this call here result in a double call to `cleanupCleanupBlobs` ?

Reply:
you're right, removed it which simplified the code at the expense of having less context, which means when we're cleaning up `l` blobs we will always keep one hour's worth of them, which I think is fine.